### ISSUE TITLE
Auto-update not working for Windows 10

### ISSUE THREAD

**Issue by diomark** [alephium/desktop-wallet#540]
**Describe the bug**
Launch alephium 1.50 wallet - it prompts for 1.5.1 update -
![image](https://user-images.githubusercontent.com/20622085/222206813-1031675e-e8f0-4ef9-ac57-be3b0810ba20.png)

but attempting to upgrade fails -
![image](https://user-images.githubusercontent.com/20622085/222207017-ad10e9c2-69a1-4192-aced-1ad77bd525b3.png)

Text of the error -
`"alephium-wallet-updater\\pending\\temp-Alephium-1.5.1.exe is not digitally signed. You cannot run this script on the current system. For more information about running scripts and setting execution policy, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170" }"`

Windows version -
Microsoft Windows [Version 10.0.19045.2604]

My powershell policies are undefined -
```
Get-ExecutionPolicy -List

        Scope ExecutionPolicy
        ----- ---------------
MachinePolicy       Undefined
   UserPolicy       Undefined
      Process       Undefined
  CurrentUser       Undefined
 LocalMachine       Undefined
```

**Comment by nop33** [comment]
Thanks a lot for the report, actively working on a fix.

**Comment by nop33** [comment]
We found a fix and release v1.5.2.

**Comment by diomark** [comment]
tested and working. tnx. 
![image](https://user-images.githubusercontent.com/20622085/222269580-13f3f843-65e6-4392-bca6-26450c9cdede.png)
